crt_portal/static/sass/custom/intake.scss

```diff
@@ -158,9 +158,21 @@
       margin-top: 0rem;
     }
 
-    .location-state--container {
+    .primary-issue {
+      width: 20rem;
+    }
+
+    .reported-reason {
+      width: 18rem;
+    }
+
+    .relevant-details {
+      width: 18rem;
+    }
```

Comment:
Note that we have known display issues with these embiggened dropdows on IE11. Noting this for the record but it shouldn't be a problem, assuming proforms staff are indeed on Edge.